src/Application.php

```diff
@@ -329,6 +329,11 @@ public function initProviders()
         $this->register(new Silex\Provider\SwiftmailerServiceProvider());
         if ($this['config']->get('general/mailoptions')) {
             $this['swiftmailer.options'] = $this['config']->get('general/mailoptions');
+        } else {
+            $this['swiftmailer.use_spool'] = false;
```

Comment:
This turns of the in-memory spooler. Not sure that is a good "default" approach, or am I missing something? :grin: